source/core/oxview.php

```diff
@@ -1017,4 +1017,60 @@ public function getSidForWidget()
 
         return $sRet;
     }
+
+    /**
+     * persparam visible, or as hidden field (in checkout)
+     * 
+     * @param $s_paramKey string
+     *
+     * @return bool
+     */
+    public function showPersParam($s_paramKey) {
+
+        $bRet = false;
+
+        if ($s_paramKey == "details")
+            $bRet = true;
+
+        return $bRet;
+    }
+
+    /**
+     * return label text for persparam (getter for text)
+     * 
+     * @param $sOxMultilangIdent
+     *
+     * @return string
+     */
+    public function getPersParamText($sOxMultilangIdent) {
+
+        if ($sOxMultilangIdent == "details") {
+            // details has different multilang-idents
+            if (isAdmin())
+                $sOxMultilangIdent = "ORDER_PACKAGE_DETAILS";
+            else
+                $sOxMultilangIdent = "LABEL";
+        }
+
+        return oxRegistry::getLang()->translateString($sOxMultilangIdent);
+    }
+
+    /**
+     * return value for persparam (getter for value)
+     * 
+     * @param $s_paramKey
+     * @param $s_paramValue
+     *
+     * @return null|other
+     */
+    public function getPersParamValue($s_paramKey, $s_paramValue) {
+
+        $_ret = null;
+
+        if ($s_paramKey == "details")
```

Comment:
How can you get a Persparam-Value with an other key then 'default'? (see also 6f3675413d67305d285e2173238a220bccc8b734)